### ISSUE TITLE
Regarding GPVAR Estimator Train Initialization GluonTS ver 5.0 

### ISSUE THREAD

**Issue by bluesky333** [awslabs/gluon-ts#825]
Hi I am trying to use GPVAR estimator for my multivariate dataset.
My data dimension is 246 and used hyperparams as below to run GPVAR estimator
after reading the paper.

```
target_dim=246#metadata["num_series"]
epochs = 2
num_layers=2
num_cells=40
batch_size=16
clip_gradient=10.0
hybridize=True
init="xavier"
learning_rate=0.001
learning_rate_decay_factor=0.5
minimum_learning_rate=5e-05
num_batches_per_epoch=50
patience=10
weight_decay=1e-08
num_parallel_samples=400
dropout_rate=0.01
cell_type='lstm'
rank=10
```
**However,** the estimator does not get trained for hours. It is stuck in initializing the optimizer
 ```
  0%|          | 0/100 [00:00<?, ?it/s]
learning rate from ``lr_scheduler`` has been overwritten by ``learning_rate`` in optimizer.
```

If I try with 2 dimension, the training is successful.

FYI,
I am not using gpu. I am using cpu.
```
use_gpu = False
ctx = mx.gpu(1) if use_gpu else mx.cpu()
prediction_length = 180
freq = "B"
start = pd.Timestamp(ts_data.columns[0], freq=freq)
metadata = {'num_series': selected_df.shape[1]-1,
                      'num_steps': selected_df.shape[0],
                      'prediction_length': prediction_length,
                      'freq': freq,
                      'start': start
                     }
```

**Comment by mbohlkeschneider** [comment]
Hi @bluesky333, 

thank you for using GluonTS. This is a known Mxnet issue with graph initialization, tracked [here](https://github.com/apache/incubator-mxnet/issues/17565). You can confirm whether this is the issue by setting `hybridize=False` as described in #607.

**Comment by bluesky333** [comment]
Hi @mbohlkeschneider 

Thank you for the prompt reply.
The graph initialization is okay after setting hybridize=False,
but training is very slow, which is sad for me as I only have CPUs.
I will close the issue for now.

**Comment by mbohlkeschneider** [comment]
Hi @bluesky333,

It might still be faster to wait for the graph initialization with `hybridize=True`. Even if graph initialization takes a while, it is probably faster overall. Note that a GPU will likely not speed the computation because the model is largely sequential and most operations cannot be parallelized.

**Comment by bluesky333** [comment]
Ok I will try with hybridize=True. Last time when I waited 4 hours it did not get initialized though.

**Comment by mbohlkeschneider** [comment]
What do you set `context_length` to? `prediction_length` of 180 is also really high. I would try reducing this.

**Comment by bluesky333** [comment]
I left `context_length` as None. For 180 `prediction_length`, I know it is high. I am thinking of reducing it to 60. Thank you for the help.

**Comment by bluesky333** [comment]
Hi @mbohlkeschneider , I tried 60 `prediction_length` and it still takes too long.
Rather than using my 246 time series as all different target-dim,
I want to use 240 time series as variable features to be used to predict 6 time series
(I am using related assets to predict certain stock asset).
I think in this way I can use all time series and make sure the run does not take so long.
But, it seems gpvar and deepvar do not use any features (except time feature) to predict.
**Is there any way for me to use time series as features to predict multivariate time series?**

**Comment by mbohlkeschneider** [comment]
Hi @bluesky333,

using related time series as the dimensions is the only way to use related time-series/dynamic features at the moment in GPVAR and DeepVAR. You can try to implement that yourself and use DeepAR for reference. Please send a pull request if you give this a shot!

**Comment by isenilov** [comment]
Hello @mbohlkeschneider 
Does it mean that features generated by `gluonts.model.deepvar._estimator.FourierDateFeatures` and other results of the transformation chain are not fed to the model?
